display_list/dl_color.h

```diff
@@ -9,10 +9,32 @@
 
 namespace flutter {
 
+enum class DlColorSpace { kSRGB = 0, kExtendedSRGB = 1, kDisplayP3 = 2 };
+
 struct DlColor {
  public:
-  constexpr DlColor() : argb_(0xFF000000) {}
-  constexpr explicit DlColor(uint32_t argb) : argb_(argb) {}
+  constexpr DlColor()
+      : alpha_(0.f),
+        red_(0.f),
+        green_(0.f),
+        blue_(0.f),
+        color_space_(DlColorSpace::kSRGB) {}
+  constexpr explicit DlColor(uint32_t argb)
+      : alpha_(((argb >> 24) & 0xff) / 255.f),
+        red_(((argb >> 16) & 0xff) / 255.f),
+        green_(((argb >> 8) & 0xff) / 255.f),
+        blue_(((argb >> 0) & 0xff) / 255.f),
+        color_space_(DlColorSpace::kSRGB) {}
+  constexpr explicit DlColor(float alpha,
```

Comment:
float -> DlScalar

Reply:
Is `explicit` required here?

Reply:
Done.  Explicit isn't required, just for single parameter constructors, removed.